package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "format": "prettier *.md scripts{,/**}/*.js --write",
     "format:check": "prettier *.md scripts{,/**}/*.js --list-different",
-    "generate": "ts-node --transpile-only ./scripts/changelog-generator.ts",
```

Comment:
What does this change do?

Reply:
This makes it possible to have scripts that specify `node` options, such as the `generate:debug` command that allows attaching a debugger. When `$_NODE_OPTS` is undefined the command is technically identical to the previous version.